### DISCUSSION TITLE
Resurrect Windows 7/8 styles taskb..

### DISCUSSION THREAD

**Discussion by TheDolphinSlayer** [valinet/ExplorerPatcher#921] [Ideas]
(since we're resurrecting start menu's now I figured it wouldn't hurt to try) restoring the OG taskbar Jumplist from Vista-Windows 8.1(I know there isn't any significance to having that feature but I mean we have the options to restore most flyouts anyways). I honestly just preferred when the row height was alot smaller than the immersive windows in windows 10/11. I know StartAllBack has them, but I doubt they've shared any information on how they personally did it themselves. Also restoring the Windows 7 Network Pane, probably hidden somewhere? Just an idea anyways.

**Comment by valinet** [comment]
Hi

Yeah, idk if those are still in there. As for how SAB does the jump lists, I have to look into it, I have no idea how that is done.

The network flyout, yeah, idk, they might have removed that from good, again, idk what was responsible for it and have no idea about how it all worked internally.
